Phoenix/hello/lib/hello_web/views/layout_view.ex

```diff
@@ -3,5 +3,8 @@ defmodule HelloWeb.LayoutView do
 
```

Comment:
@samu975 what does a view do?

Reply:
Views are modules responsible for rendering. handles the structures data from the controller and converts it to a presentation to be shown to user